probert/utils.py

```diff
@@ -42,29 +42,37 @@ def _clean_env(env):
     return env
 
 
-def _log_stream(stream, name):
+def _decode_stream(stream, name):
     if stream:
+        try:
+            log_input = text = stream.decode('utf-8')
+        except UnicodeDecodeError as ude:
+            log.debug(f'UnicodeDecodeError on {name}: {ude}')
+            text = None
+            log_input = stream.decode('utf-8', 'backslashreplace')
         log.debug(f'{name}: ------------------------------------------')
-        for line in stream.splitlines():
+        for line in log_input.splitlines():
             log.debug(line)
+        return text
```

Comment:
if we still return `None` here in case of UnicodeDecodeError, we don't solve the issue with the sizing information, do we?